### PR TITLE
added data structures to glossary

### PR DESCRIPTION
**Pull request by coderJakub**

### CODE REVIEW

**Review by henrikalbrecht — APPROVED** [review]

File: docs/architecture/architecture_notebook.adoc [hunk 1]
```diff
@@ -64,6 +64,21 @@ https://www2.htw-dresden.de/~anke/openup/core.tech.common.extend_supp/guidances/
 
 - _Klasse_ PG: Die Klasse speichert alle relevanten Informationen zu einer Projektgruppe welche eine VM beantragt haben.
 
+- _Klasse_ Anmeldedaten:: Diese Klasse speichert die Zusammensetzung von Benutzername und Passwort. Die gültige Emailadresse muss das `@`-Zeichen und einen `.` Punkt enthalten. Alias: Login.
```
Comment (don-ny) [inline docs/architecture/architecture_notebook.adoc:67]:
```suggestion
- _Klasse_ Anmeldedaten:: Diese Klasse speichert die Zusammensetzung von Benutzername und Passwort. Die gültige Emailadresse muss das `@`-Zeichen und mindestens einen `.` Punkt enthalten. Alias: Login.
```

File: docs/architecture/architecture_notebook.adoc [hunk 2]
```diff
@@ -64,6 +64,21 @@ https://www2.htw-dresden.de/~anke/openup/core.tech.common.extend_supp/guidances/
...
+
+
+- _Klasse_ VM-Daten:: Diese Klasse speichert alle relevanten Informationen zu einer virtuellen Maschine. Die Daten umfassen Hostname, IP, Status und für vergebene VMs auch den Projektnamen, die Kontaktdaten, das OS-Config-Template und das Ausstellungsdatum. Datentypen: String, int, date, boolean, email, IP. Gültige Emailadresse und IPv4-Adresse.
+
+- _Klasse_ Email-Template:: Diese Klasse dient als Vorlage für eine Email mit Betreff und Inhalt sowie den dazugehörigen Namen.
```
Comment (don-ny) [inline docs/architecture/architecture_notebook.adoc:72]:
```suggestion
- _Klasse_ Email-Template:: Diese Klasse dient als Vorlage für eine Email mit Betreff und Inhalt, sowie den dazugehörigen Namen.
```

File: docs/requirements/glossary.adoc [hunk 3]
```diff
@@ -73,9 +73,28 @@ In diesem Dokument werden die wesentlichen Begriffe aus dem Anwendungsgebiet (Fa
 | Emailadresse muss `@`-Zeichen und `.` Punkt enthalten.
 | Login
 
-| xx
-| xx
-| xx
-| xx
-| xx
+| VM-Daten
+| Sammlung aller Daten einer virtuellen Maschine. Die Daten umfassen hostname, ip, status sowie für vergebene VMs auch den Projektnamen, die Kontaktdaten, das OS-Config-Template und das Ausstellungdatum.
```
Comment (don-ny) [inline docs/requirements/glossary.adoc:77]:
```suggestion
| Sammlung aller Daten einer virtuellen Maschine. Die Daten umfassen Hostname, IPv4-Adresse, Status, sowie für vergebene VMs auch den Projektnamen, die Kontaktdaten, das OS-Config-Template und das Ausstellungdatum.
```

**Review by don-ny — CHANGES_REQUESTED** [review]
- see individual changes and...
...
- change glossary entry name "IP Adresse" to "IP-Adresse" (line 33)
- add glossary abbreviation: IP
- add glossary abbreviation: IPv4

**Pull request merged**